src/gui/properties/proplistdelegate.h

```diff
@@ -48,25 +49,27 @@ enum PropColumn
     AVAILABILITY
 };
 
-class PropListDelegate final : public ProgressBarDelegate
+class PropListDelegate final : public QStyledItemDelegate
 {
     Q_OBJECT
+    Q_DISABLE_COPY(PropListDelegate)
 
 public:
     explicit PropListDelegate(PropertiesWidget *properties);
 
-    void setEditorData(QWidget *editor, const QModelIndex &index) const override;
-    QWidget *createEditor(QWidget *parent, const QStyleOptionViewItem & /* option */, const QModelIndex &index) const override;
-
 public slots:
     void setModelData(QWidget *editor, QAbstractItemModel *model, const QModelIndex &index) const override;
-    void updateEditorGeometry(QWidget *editor, const QStyleOptionViewItem &option, const QModelIndex & /* index */) const override;
+    void updateEditorGeometry(QWidget *editor, const QStyleOptionViewItem &option, const QModelIndex &) const override;
```

Comment:
Actually you can have parameter name here but omit it in method definition.

Reply:
Done.